### ISSUE TITLE
Exception for inaccessible attribute

### ISSUE THREAD

**Issue by andrewpaulstone** [jqno/equalsverifier#753]
**Describe the bug**
If one of the attributes on the class to be tested is of type PrintStream, initialised with System.out, then the equals verifier fails with the exception shown below.

**To Reproduce**
Create a class with an attribute initialised as above and test it using the equals verifier.

**Code that triggers the behavior**

Class under test:

import java.io.PrintStream;
import java.util.Objects;

public class Failing {

    private final PrintStream outputStream;

    private final String something;

    public Failing(final PrintStream outputStream,
                   final String something) {
        this.outputStream = outputStream;
        this.something = something;
    }

    @Override
    public boolean equals(final Object pOther) {
        if (this == pOther) {
            return true;
        }

        if (pOther == null) {
            return false;
        }

        if (getClass() != pOther.getClass()) {
            return false;
        }

        final Failing failing = (Failing) pOther;
        return Objects.equals(something, failing.something);
    }

    @Override
    public int hashCode() {
        return Objects.hash(something);
    }

}

test:

import nl.jqno.equalsverifier.EqualsVerifier;
import org.junit.jupiter.api.Test;

class FailingTest {

    @Test
    public void testEqualsAndHash() {
        EqualsVerifier.forClass(Failing.class).verify();
    }

}

**Error message**
java.lang.AssertionError: EqualsVerifier found a problem in class <test class>.
-> Unable to make field private char[] java.io.Writer.writeBuffer accessible: module java.base does not "opens java.io" to unnamed module @2d38eb89

**Expected behavior**
An exception should not be raised.
A viable option could be to simply ignore any attributes that are inaccessible and simply report a warning.

**Version**
What version of EqualsVerifier are you using?
3.12.3

**Comment by jqno** [comment]
I'll add a fix for that as soon as I have some time.

In the mean time, you can add prefab values:

```
EqualsVerifier.forClass(Failing.class)
    .withPrefabValues(PrintStream.class, System.out, System.err)
    .verify();
```

**Comment by jqno** [comment]
Fixed in version 3.12.4.

Note that for this release, I've changed the release process. If you notice any problems, please let me know!
